CodeGeneration/Sources/generate-swiftbasicformat/BasicFormatFile.swift

```diff
@@ -136,8 +136,21 @@ let basicFormatFile = SourceFile {
       }
     }
 
-    FunctionDecl("open func requiresTrailingSpace(_ tokenKind: TokenKind) -> Bool") {
-      SwitchStmt(expression: Expr("tokenKind")) {
+    FunctionDecl("open func requiresTrailingSpace(_ token: TokenSyntax) -> Bool") {
+      SwitchStmt("""
+        switch (token.tokenKind, token.nextToken(viewMode: .sourceAccurate)?.tokenKind) {
+        case (.asKeyword, .exclamationMark),
+             (.asKeyword, .postfixQuestionMark),
+             (.initKeyword, .postfixQuestionMark),
+             (.tryKeyword, .exclamationMark),
```

Comment:
Also added for `try!`

Reply:
Should `(.initKeyword, .leftParen)` be in the list to produce `init()` without the additional whitespace?

Reply:
Yes. If you want to open a pull request for it, that would be a very worthwhile addition.